packages/core/src/address/address.ts

```diff
@@ -21,4 +21,5 @@ export interface AddressRequestBody {
         fieldId: string;
         fieldValue: string | number | string[];
     }>;
+    email?: string;
```

Comment:
Can email be required in this interface?

Reply:
fixed

Reply:
What did you fixed?

Reply:
This interface used in many places, in some of them email is not required or doesn't exist